config/ember-try.js

```diff
@@ -6,10 +6,10 @@ module.exports = async function() {
   return {
     scenarios: [
       {
-        name: 'ember-lts-3.12',
+        name: 'ember-3.13',
         npm: {
           devDependencies: {
-            'ember-source': '~3.12.0'
+            'ember-source': '~3.13.0'
```

Comment:
Probably shouldn't change this entry, could add a new one for 3.13 though

Reply:
I think its fine, since I updated the README to say min supported version is 3.13. 
Doesn't make sense to test on a version that you know won't work

Reply:
👍 I think it was in a weird state when I looked at it with ignoring 3.12 failures in travis, but removed 3.12 here